src/frontend/test_runner/tests/gen/testcases.rs

```diff
@@ -0,0 +1,289 @@
+// Copyright 2022 Singularity Data
```

Comment:
Is this file auto-generated?

Reply:
I added because license-eye fail it because didn,t have that Copyright. See the conversion from github-actions.

Reply:
I mean the whole file seems to be automatically generated, `gen` as the path name. 😄

Reply:
I not sure le. What should I do here?

Reply:
This file is no longer useful and should not be added here.
It was a result of #2917: This file used to be generated and ignored, but no longer generated after the refactor.

Reply:
Ok, So I should delete it right? include the directory gen.

Reply:
Yes delete it. (btw git does not track empty directories. So removing directory is optional - but why not)